### PR TITLE
Сheck adapter from infection.json config file in order to install a package

### PR DESCRIPTION
**Pull request by maks-rafalko**
Followup for https://github.com/infection/infection/pull/1102

In #1102, we added a feature to ask a confirmation and automatically install adapter's package when user runs Infection like `--test-framework=phpspec`

This PR additionally checks `infection.json` to trigger that logic.

### CODE REVIEW

File: src/Command/InfectionCommand.php [hunk 1]
```diff
@@ -353,10 +354,15 @@ private function runConfigurationCommand(Locator $locator): void
 
     private function installTestFrameworkIfNeeded(InputInterface $input, OutputInterface $output): void
     {
-        $container = $this->getApplication()->getContainer();
+        $installationDecider = $this->container->getAdapterInstallationDecider();
+
+        try {
+            $configTestFramework = $this->container->getConfiguration()->getTestFramework();
+        } catch (FileNotFound $e) {
```
Comment (maks-rafalko) [inline src/Command/InfectionCommand.php]: When Infection is run for the first time, there is no `infection.json` file yet

**Comment by maks-rafalko** [comment]
ping @infection/core

File: src/Command/InfectionCommand.php (hunk 1 above)
Comment (theofidry) [reply src/Command/InfectionCommand.php]: Shouldn't it be run _after_ the `runConfigurationCommand()` then?

Comment (maks-rafalko) [reply src/Command/InfectionCommand.php]: 👍 nice, updated

**Comment by maks-rafalko** [comment]
Thank you @theofidry

**Pull request merged**